lib/elixir/lib/system.ex

```diff
@@ -114,6 +114,11 @@ defmodule System do
     end
   end
 
+  # Returns OTP version that Elixir was compiled with.
+  defmacrop get_otp_version do
```

Comment:
Perhaps we should change that output, considering that we have function named `System.otp_release/0`.